src/client/Statistics.ts

```diff
@@ -20,7 +20,50 @@ import {
  * first would be .2*2=.4
  */
 export function calculateAverages(responseData: GetFullScoresResponseData): SubmissionAverageStatistics {
-  throw new Error("Not Yet Implemented");
+  let cloned = [...responseData.assignments];
```

Comment:
What are you trying to do by cloning `assignments`? If you're trying to make a copy to modify things you need to make a deep copy, but I'm not sure how making a copy helps you.